src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -55,7 +55,10 @@
      * - 'throw' remove blank lines after a line with a 'throw' statement
      * - 'use' remove blank lines between 'use' import statements
      * - 'useTrait' remove blank lines between 'use' trait statements
-     * - 'curly_brace_open' remove blank lines after a curly opening brace ('{')
+     * Blocks:
```

Comment:
please remove that line, it looks like we have one key like `options` and second like `blocks`, while we have only one options ;)